tests/pods_test.go

```diff
@@ -47,68 +55,74 @@ func TestPod(testT *testing.T) {
 		deploymentID := getDeploymentID(retryT, kPod.GetName())
 
 		podCount := getPodCount(retryT, deploymentID)
-		log.Infof("Pod count: %d", podCount)
+		testT.Logf("Pod count: %d", podCount)
 		require.Equal(retryT, 1, podCount)
 
 		// Get the test pod.
 		pods := getPods(retryT, deploymentID)
-		log.Infof("Num pods: %d", len(pods))
+		testT.Logf("Num pods: %d", len(pods))
 		require.Len(retryT, pods, 1)
 		pod := pods[0]
 
-		log.Infof("Pod: %+v", pod)
+		testT.Logf("Pod: %+v", pod)
 
 		// Verify the container count.
 		require.Equal(retryT, int32(2), pod.ContainerCount)
 
-		// Verify the events.
-		var loopCount int
-		var events []Event
-		for {
-			events = getEvents(retryT, pod)
-			log.Infof("%d: Events: %+v", loopCount, events)
-			if len(events) == 4 {
-				break
+		if os.Getenv("COLLECTION_METHOD") == "NO_COLLECTION" {
+			testT.Logf("Skipping parts of TestPod that relate to events because env var \"COLLECTION_METHOD\" is " +
+				"set to \"NO_COLLECTION\"\n. This is expected to only happen when Sensor version is 3.74.x " +
+				"(support exception / compatibility tests)\"")
```

Comment:
I would make this log shorter and make sure it is correct.

> This is expected to only happen when Sensor version is 3.74.x
Nope, you can set NO_COLLECTION also for other Sensor versions.

```suggestion
		// NO_COLLECTION is the only supported collection mode for 3.74
		testT.Log("Skipping parts of the test because the COLLECTION_METHOD in Collector is set to \"NO_COLLECTION\"")
		// Using NO_COLLECTION as the collection method results in no Collector container being deployed
		// and no process indicators being emitted. Thus, there is no point in asserting that Sensor or Central 
		// is aware of those processes.
```

Reply:
I made the assumption here that we would only set `NO_COLLECTION` for a support exception like this but I guess that is not strictly correct. I've removed that part of the comment.